http-clients/src/main/java/com/palantir/remoting/http/errors/SerializableErrorErrorDecoder.java

```diff
@@ -101,18 +106,16 @@ public Exception decode(String methodKey, Response response) {
         }
     }
 
-    private static Exception constructException(Class<? extends Exception> exceptionClass, String message) {
-        try {
-            return exceptionClass.getConstructor(String.class).newInstance(message);
-        } catch (Exception e1) {
-            return new RuntimeException(String.format(
-                    "Failed to construction exception of type %s, constructing RuntimeException instead: %s%n%s",
-                    exceptionClass.toString(), e1.toString(), message));
+    private static Exception constructException(
```

Comment:
Not yet! Server side fix is merged (so you could snapshot). On https://github.com/palantir/http-remoting/pull/35, @uschi2000 said that we'll likely release 0.7.0 once my fixes go in.